cargo-marker/Cargo.toml

```diff
@@ -17,10 +18,10 @@ name = "cargo-marker"
 path = "src/main.rs"
 
 [dependencies]
-camino         = { version = "1.1", features = ["serde1"] }
-cargo_metadata = "0.15.4"
-clap           = { version = "4.0", features = ["string", "derive"] }
-once_cell      = "1.17.0"
-serde          = { version = "1.0", features = ["derive"] }
-serde_json     = "1.0"
-toml           = "0.7"
+camino.workspace         = true
```

Comment:
I suggest using `key = { worspace = true }` instead.
This way it will align nicer.

Compare this code in the diff with this:

```toml
camino         =  { workspace = true }
cargo_metadata =  { workspace = true }
clap           =  { workspace = true }
once_cell      =  { workspace = true }
serde          =  { workspace = true }
serde_json     =  { workspace = true }
toml           =  { workspace = true }
```
Everything is now aligned with `taplo`, and the word `.workspace` doesn't clutter the package name